src/portable/portablectl.c

```diff
@@ -926,6 +1020,9 @@ static int help(int argc, char *argv[], void *userdata) {
                "  read-only NAME|PATH [BOOL]  Mark or unmark portable service image read-only\n"
                "  remove NAME|PATH...         Remove a portable service image\n"
                "  set-limit [NAME|PATH]       Set image or pool size limit (disk quota)\n"
+               "  upgrade NAME|PATH [PREFIX...]\n"
+               "                              Upgrades and restarts the specified portable\n"
+               "                              service image\n"
```

Comment:
i'd move this up to where attach/detach is listed, it has nmore to to with that.

(this is ordered alphabetically only by accident, we don't actually intend to order things alphabetically in --help texts, but group things by context)

Reply:
moved